pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -285,16 +343,39 @@ func (c *clustersReconciler) updateStatus(ctx context.Context, cluster *kueuealp
 	return c.localClient.Status().Update(ctx, cluster)
 }
 
+func (c *clustersReconciler) runGC(ctx context.Context) {
+	log := ctrl.LoggerFrom(ctx)
```

Comment:
```suggestion
	log := ctrl.LoggerFrom(ctx).WithName("MultiKueueGC")
```

Then you can simplify some messages in the rest of the code.

Reply:
done